packages/components/README.md

```diff
@@ -19,6 +19,11 @@ test  :
   npm run test
   ```
 
```

Comment:
Let's maintain the spacing format of the script list

Reply:
I did not get this. What did you mean by spacing format?